scripts/check_referenced_variables.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+attributes_file_path="https://raw.githubusercontent.com/pagopa/io-functions-template/add_function_json_check/scripts/attributes.json"
```

Comment:
We can refer dynamically the branch name with something like
```suggestion
brach_name=$(git branch | sed -n -e 's/^\* \(.*\)/\1/p')
attributes_file_path="https://raw.githubusercontent.com/pagopa/io-functions-template/${brach_name}/scripts/attributes.json"
```

Reply:
Do you mean that all the repo has a settings file not referred because the template version is? I thinks this is counterintuitive. If we want a global check for all the repos we should consider to add these steps into a pipeline module and not into the repo codebase.

Reply:
Refer the current working dir allow to check PR that modify the `settings.json` file

Reply:
The idea was to check attributes against an updated list, defined somewhere (I choosed fn-template, bad choice I admit). 
But could be good enough to preparare the full list of bindings attributes right now and hope they don't change over time.

Reply:
@BurnedMarshal @balanza I slightly modified the script this way: it checks values using the local file, but I added a new script to eventually override the `attribute.json` file [35a692c](https://github.com/pagopa/io-functions-template/pull/45/commits/35a692c1be1198d0a10ff6b828e8b9de56e8d307)